fabric-lifecycle-events-v1/README.md

```diff
@@ -0,0 +1,206 @@
+# Fabric Lifecycle Events (V1)
+
+## Organization
+
+Events in this module belong to a single class which relates to the category of events.
+The implementation of event callbacks are nested classes.
+
+For example, `ServerLifecycleEvents.SERVER_STARTING` is in `ServerLifecycleEvents` and the implementation of the event's callback would implement `ServerLifecycleEvents.ServerStarting`.
+
+# Server Events
+
+<details><summary>Events related to objects on a logical server.</summary>
+
+## `ServerLifecycleEvents`
+
+<details><summary>Contains lifecycle events related to a Minecraft Server.</summary>
+
+#### `ServerLifecycleEvents.SERVER_STARTING`
+
+- Called when a Minecraft server is starting. This is called before any worlds are loaded, no players are connected yet and the `PlayerManager` is null.
+- Mods which rely on tracking the current server instance may use this event to capture the server instance.
+
+#### `ServerLifecycleEvents.SERVER_STARTED`
+
+- Called when a Minecraft server has finshed starting and is about to tick for the first time. When this event is called, all worlds have been loaded and players are ready to be accepted.
+
+#### `ServerLifecycleEvents.SERVER_STOPPING`
+
+- Called when a Minecraft server has started shutting down.
+- This occurs before the server's network channel is closed and before any connected players are disconnected.
+
+#### `ServerLifecycleEvents.SERVER_STOPPED`
+
+- Called when a Minecraft server has stopped.
+All worlds have been closed and all (block) entities and players have been unloaded.
+- If a mod stores the server instance anywhere, this event should be used for reference cleanup.
+- Depending on the implementation of the server the following may occur after this event is called:
+    - On an integrated server (`EnvType.CLIENT`), the client will continue running.
+    - On a dedicated server (`EnvType.SERVER`), this will be the last event called. The JVM will terminate after this event.
+
+#### `ServerLifecycleEvents.START_DATA_PACK_RELOAD`
+- Called before a Minecraft server reloads data packs.
+- In vanilla, this is typically called after `/reload` command is executed.
+
+#### `ServerLifecycleEvents.END_DATA_PACK_RELOAD`
+- Called after a Minecraft server has reloaded data packs.
+- This event does not garuntee the data pack reload was successful.
+- Per the `EndDataPackReload` interface, the last `boolean` parameter specifies whether the data pack reload was successful.
```

Comment:
Is `boolean` the actual parameter name? If not, it should either not be formatted or use the actual parameter name.